packages/react-native-reanimated/Common/cpp/worklets/Tools/JSScheduler.cpp

```diff
@@ -27,9 +27,12 @@ JSScheduler::JSScheduler(
       }),
       rnRuntime_(rnRuntime),
       runtimeExecutor_(runtimeExecutor) {}
-#endif // REACT_NATIVE_MINOR_VERSION >= 74 && defined(RCT_NEW_ARCH_ENABLED
+#endif // REACT_NATIVE_MINOR_VERSION >= 74 && defined(RCT_NEW_ARCH_ENABLED)
 
 const std::shared_ptr<CallInvoker> JSScheduler::getJSCallInvoker() const {
+#if REACT_NATIVE_MINOR_VERSION >= 74 && defined(RCT_NEW_ARCH_ENABLED)
+  return nullptr;
```

Comment:
This is temporary or not? 🤔

Reply:
When I tested it on

- #6378

it worked correctly. JSCallInvoker is needed for TurboModule constructor:
![Screenshot 2024-09-24 at 13 09 10](https://github.com/user-attachments/assets/995d9af9-ec64-49dd-8159-06e74addb1d1)

When I tested it it seemed as nullptr was okay there. @WoLewicki can you give some insight here?

Reply:
Actually it's not necessarry in this PR. I'll revert.